packages/validator/src/commands/convertCmd.ts

```diff
@@ -1,5 +1,6 @@
 import fs from 'fs';
```

Comment:
Wouldn't it be better to use [`command`](https://github.com/yargs/yargs/blob/master/docs/api.md#commandmodule) and have the `convert` and `validate` CLIs as one CLI with two different commands (the same way as deployer's `deploy` and `remove` are modeled)?

Reply:
That makes sense in deployer, however for users just running validations and conversions im trying to differentiate those 2 programs as much as possible so there is no confusion in what to expect from the command. It simply makes more sense to validate specifications with validator and convert them in convertor instead of making both validations and conversions in validator...